modules/uhttp/http.go

```diff
@@ -78,7 +78,7 @@ type Config struct {
 	modules.ModuleConfig
 	Port    int           `yaml:"port"`
 	Timeout time.Duration `yaml:"timeout"`
-	Debug   *bool         `yaml:"debug"`
+	Debug   bool          `yaml:"debug"`
```

Comment:
Thanks for the context. I believe there are use cases for that. But when we are not using `did not specify` has different behavior with `false`, let's just use basic `bool` instead of `*bool`.